src/contrib/cluster/Akka.DistributedData/reference.conf

```diff
@@ -98,4 +98,36 @@ akka.actor {
 	"Akka.DistributedData.Serialization.ReplicatedDataSerializer, Akka.DistributedData" = 11
     "Akka.DistributedData.Serialization.ReplicatorMessageSerializer, Akka.DistributedData" = 12
   }
+  serialization-settings {
+    akka-replicated-data {
+        mappings {
+            "System.Object" = 0
```

Comment:
This is a general idea to keep things working. While this design works with mostly the same protobuf definitions as core akka ddata ones, there was a serious problem around notion of generics: protobuf doesn't support generic types. In akka it's not the problem, as JVM erases them anyway and their solution is basically *fuck it, just assume they don't exist*.

This however is not the case in .NET - we need to know all generic type parameters before we instantiate generic types using them. My solution here was to add extra `uint32` fields in protobuf definitions where they were necessary - those fields provide basic tagging of core types. 

Since we may need to declare tagging for types that are defined in assemblies, users don't have control over, I've decided to move them to be defined in HOCON as you can see here.